loading_NYC_taxi_data.ipynb

```diff
@@ -0,0 +1,854 @@
+{
```

Comment:
<p>This is very interesting to see! There's a sharp drop in trips lockdown started. It might be useful to add some markers (vertical lines) indicating various stages in the pandemic to make that more clear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/harrislam1/covid19_NYC_taxi_trips/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>